examples/platforms/nrf52811/README.md

```diff
@@ -222,29 +216,106 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
 
```

Comment:
Break down the step 8 in Running the example into 3 steps:

8. Open a terminal connection on the second board and attach a node to the network.
   The CLI MTD example uses the direct UART connection.
9. Start a terminal emulator like screen and connect to the used COM port with the following UART settings:
    - Baud rate: 115200
    - 8 data bits
    - 1 stop bit
    - No parity
    - HW flow control: RTS/CTS
This allows you to view the raw UART output.
10. Run the following command to connect to the second board.
 ```bash
 screen /dev/ttyACM1 115200
 ```
Now you are connected with the CLI.

Reply:
Thanks. I will combine step 9 and 10 since they are about the same.

Reply:
They are clearly two different steps, so this would basically contradict the reason behind my edit. Whenever we use steps, we go for "1 step = 1 action", which also means "1 action = 1 imperative verb".

Reply:
Yes but step 10 is just execution of description in stage 9. So they should be combined, right?

Reply:
OK, it's one huge step then. What about:

8. Open a terminal connection on the second board and attach a node to the network:
   a. Start a terminal emulator like screen.  
   b. Connect to the used COM port with the following direct UART settings:
   * Baud rate: 115200
   * 8 data bits
   * 1 stop bit
   * No parity
   * HW flow control: RTS/CTS
     This allows you to view the raw UART output.
   c. Run the following command to connect to the second board.
```shell
screen /dev/ttyACM1 115200
```
You are now connected with the CLI.